### DISCUSSION TITLE
Bot is not playing anything

### DISCUSSION THREAD

**Discussion by oliverottis3** [jagrosh/MusicBot#1560] [Troubleshooting]
I have downloaded the correct version of java. The bot joins the channel but its not playing. 0.3.9 works. Plis help!

Here is the console:
![JMusicBot 28 4 2024 16 39 43](https://github.com/jagrosh/MusicBot/assets/123867158/bcd52019-6f40-44f9-a525-150b49d14f34)

**Comment by YoneWolf** [comment]
I have the same exact issue

**Comment by Slimee44** [comment]
<img width="417" alt="Screenshot 2024-05-10 at 2 44 19 PM" src="https://github.com/jagrosh/MusicBot/assets/169414938/798950ef-484f-4f91-9936-4f068ae7eb82">
 This is what bounces back for me.  Additionally, the bot joins the voice channel, disconnects, rejoins, disconnects, rejoins, rinse/repeat.  At no point does it play music.  Bot worked fine on 5/8 and stopped on 5/9.  0.3.9 doesnt work either, it bounces back an even longer error, but if I try and run my OTHER 0.3.9 bot on another computer where I never installed Java 11, it runs fine...  But on this machine... even if I get rid of Java 11 and go back to 8, it still doesn't work.
